humanguide/layout/05-dialogues.rst

```diff
@@ -0,0 +1,24 @@
+Dialogues
+=========
+
+A dialogue is a way of informing the user about important notices or
+letting them to complete easy single action on a current page.
+
+Dialogues should be clear and concise. Using one color accent for the
+main action (red for destructive actions, green for positive actions,
+grey for neutral actions) and avoid ambiguous button text.
```

Comment:
This statement is unclear to me. Are you trying to say that only one action should be possible from a single dialog? In that case, maybe this is more appropriate:

"A dialog should only have one action which the user may confirm or deny. This action may be positive, negative, or neutral. Positive actions add something to their subjects. Negative actions remove or delete something. Neutral actions do not add or remove from the subject, but still make a change.

"The button which confirms the action should be accented in red if the action is destructive, green if it is positive, and blue if it is neutral. The other options, which should not make changes, should be gray."

Reply:
Rephrase time. The idea is that no matter how many options we have in a dialog, only one must be predominant and therefore colored. What about:

«Dialogs should be unambiguous and concise in its wording and coloring only the most common action. This common action or main action should be place as the top button in vertical dialogs or to the right in horizontal dialogs.

Main actions may be positive, negative, or neutral. Positive actions add something to their subjects. Negative actions remove, delete something or are changes that can't be undone. Neutral actions do not add or remove from the subject, but still make a change.

The button which confirms the action should be accented in red if the action is destructive, green if it is positive, and blue if it is neutral. The other options should be gray.

The text in a button should resume its action avoiding yes/no questions.»

Reply:
Sounds better. Copyedited, that's:

"Dialogs should be unambiguous and concise in their wording. Only the most common, or main, action should be highlighted with color. The main action should be placed topmost in vertical dialogs or rightmost in horizontal dialogs.

"Main actions may be positive, negative, or neutral. Positive actions add something to their subjects. Negative actions remove, delete, or otherwise make changes that can't be easily undone. Neutral actions do not add or remove from the subject, but still make a change.

"The main action should be highlighted in red if it is destructive, green if it is positive, or blue if it is neutral. The corresponding theme values for these are `theme.whatever`, ... or `theme.whatever`. The remaining options should be gray, or `theme.whatever`.

"Dialogs should *resume* actions, not ask a yes or no question. For example, the user has selected an option to delete a contact. Your dialog should ask, "Are you sure you would like to delete this contact?" The options should then be "Delete", a negative action colored in red, and "Cancel", colored in gray."

*consider adding a photo illustrating this example here*

Reply:
We may not even want to prescribe colors in this section, instead giving the theme values for each color. Themes could have different colors for different types of actions, after all..

Reply:
I dreamt with that. I forgot to add a section about the use of the themes